Makefile

```
@@ -1,3 +1,16 @@
+# ------------------------------------------------------------------------------
+# Configuration - Repository
+# ------------------------------------------------------------------------------
+
+REPO ?= github.com/kong/gateway-operator
+REPO_INFO ?= $(shell git config --get remote.origin.url)
+# TODO: remove --always after first tag
+TAG ?= $(shell git describe --tags --always)
```

We should be able to just leave this in its final form, correct? The first proper image we ship should be from a tagged commit, so we don't really need to worry about the current limbo period.
```suggestion
TAG ?= $(shell git describe --tags --always)
```
Worst case, this is just outputting nothing to stdout at present, so any images we do build before then just get an empty tag and show up in stats as an unknown version, which isn't the end of the world (it's arguably correct!).

---

😐 well.... allllllllllllright but your suggestion doesn't reflect what I think you wanted to convey :)